pysrc/scripts/sampling.py

```diff
@@ -81,6 +94,37 @@ def clean_results(results_folder: str) -> None:
     for tmp_file in result_files:
         os.remove(os.path.join(os.getcwd(), results_folder, tmp_file))
 
+def get_File_Combinations(samples, config):
+    data_type_dict = {}
+    for num_files in range(0, len(samples)):
+        for sam_file in range(0, len(samples[num_files])):
+            if config.header:
+                particular_line = linecache.getline(samples[num_files][sam_file][0], 1)
+            else:
+                #TODO Get Next Lines if it was empty
+                particular_line = linecache.getline(samples[num_files][sam_file][0], 2)
+            particular_line = particular_line.split('\n')
```

Comment:
I think you can use a parameter to `split` that only returns the first split part. This could massively reduce the workload as we're not using the other parts of the split.

Reply:
The parameter is called `maxsplit`. See <https://docs.python.org/3.3/library/stdtypes.html#str.split>.